Jucer2CMake/src/main.cpp

```diff
@@ -181,6 +182,19 @@ juce::String makeValidIdentifier(juce::String s)
 }
 
 
+// Matches _FRUT_make_valid_configuration_name in Reprojucer.cmake.
+juce::String makeValidConfigurationName(const juce::String& configurationName)
+{
+  auto validName = configurationName.toStdString();
+
+  validName = std::regex_replace(validName, std::regex{"[^A-Za-z0-9_]+"}, " ");
```

Comment:
maybe extract regex already duplicated once ?

Reply:
`^[A-Za-z0-9_]+$` and `[^A-Za-z0-9_]+` are not the same.